libpay-stripe/src/main/scala/com/wix/pay/stripe/ChargeMetadataToEmailHelper.scala

```diff
@@ -0,0 +1,87 @@
+package com.wix.pay.stripe
+
+import com.wix.pay.creditcard.CreditCard
+import com.wix.pay.model.{Customer, Deal, OrderItem}
+import com.wix.pay.stripe.ChargeMetadataToEmailHelper._
+
+import scala.collection.immutable.ListMap
+
+class ChargeMetadataToEmailHelper {
+  def getMetadataForEmail(creditCard: CreditCard, customer: Option[Customer], deal: Option[Deal]): Map[String, String] = {
+
+    val customerInfo = ListMap(
+      "Billing Address" → billingAddressInfo(creditCard),
+      "Customer Name" → customer.flatMap(customerNameInfo),
+      "Customer Phone" → customer.flatMap(_.phone),
+      "Customer Email" → customer.flatMap(_.email),
+      "Customer IP" → customer.flatMap(_.ipAddress),
+      "Invoice Id" → deal.flatMap(_.invoiceId),
+      "Shipping Address" → deal.flatMap(shippingAddressInfo)
+    )
+
+    val includedChargesInfo = ListMap(
+      "Included Charges: Tax" → deal.flatMap(chargeTaxInfo),
+      "Included Charges: Shipping" → deal.flatMap(chargeShippingInfo)
+    )
+
+    val itemsInfo = orderItemsInfo(deal)
+
+    val emailPairs = (customerInfo.toSeq ++ itemsInfo.toSeq ++ includedChargesInfo.toSeq).collect {
```

Comment:
Stripe documentation says: "Note: You can specify up to 20 keys, with key names up to 40 characters long and values up to 500 characters long." Please add validation for keys number.